### PR TITLE
[Bugfix: PDFAnnotator] TA grading impacts student view

### LINKED ISSUE #5122: pdf rotation cookie from TA grading impacts student view
**Describe the bug**
If I'm a grader in course A, and I use the TA grading interface to view and grade and annotate PDF files.  If the last time I used the TA grading interface I rotated a pdf, that setting is saved in a cookie.

If I'm just a student in course B, and I view the annotations on a pdf I've submitted for that course, the cookie setting for TA grading seems to carry over into the viewer.  from the student side I cannot control this rotation setting.

**Expected behavior**
Either introduce the rotation option on the student view of annotated PDFs, or just ignore the cookie setting for rotation and don't let it rotate the student view

### PR DESCRIPTION
**Pull request by mukul-kmr-jha**
### Please check if the PR fulfills these requirements:

* [x] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)
* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
See detailed description in this [issue](https://github.com/Submitty/Submitty/issues/5122#issue-574416623)

### What is the new behavior?
Closes #5122 
Now in the student view of the PDF , cookie values are ignored and its just shows the plain PDF without any rotation and zoom.

### CODE REVIEW

File: site/public/js/pdf/PDFAnnotateEmbedded.js [hunk 1]
```diff
@@ -7,13 +7,7 @@ var documentId = '';
 var PAGE_HEIGHT;
 var NUM_PAGES = 0;
 
-window.RENDER_OPTIONS = {
-    documentId,
-    userId: "",
-    pdfDocument: null,
-    scale: parseFloat(localStorage.getItem('scale')) || 1,
-    rotate: parseInt(localStorage.getItem('rotate')) || 0
-};
```
Comment (MasterOdin) [inline site/public/js/pdf/PDFAnnotateEmbedded.js]: Let's leave this as a global object, and add a new key/value pair `studentPopup: false`.

File: site/public/js/pdf/PDFAnnotateEmbedded.js [hunk 2]
```diff
@@ -27,18 +21,25 @@ pdfjsLib.GlobalWorkerOptions.workerSrc = 'vendor/pdfjs/pdf.worker.min.js';
 
 //For the student popup window, buildURL doesn't work because the context switched. Therefore, we need to pass in the url
 //as a parameter.
-function render_student(gradeable_id, user_id, file_name, file_path, pdf_url) {
-    render(gradeable_id, user_id, "", file_name, file_path, 1, pdf_url)
+function render_student(gradeable_id, user_id, file_name, file_path, pdf_url, isStudent) {
+    render(gradeable_id, user_id, "", file_name, file_path, 1, pdf_url, isStudent )
```
Comment (MasterOdin) [inline site/public/js/pdf/PDFAnnotateEmbedded.js]: In this function, let's set:
```
window.RENDER_OPTIONS.scale = 1;
window.RENDER_OPTIONS.rotate = 0;
window.RENDER_OPTIONS.studentPopup = true;
```

and revert the changes you've made.

File: site/public/js/pdf/PDFAnnotateEmbedded.js [hunk 3]
```diff
@@ -27,18 +21,25 @@ pdfjsLib.GlobalWorkerOptions.workerSrc = 'vendor/pdfjs/pdf.worker.min.js';
...
         file_path: file_path
-    }
+    };
+  window.RENDER_OPTIONS = {
+    documentId,
+    userId: "",
+    pdfDocument: null,
+    scale: !isStudent ? parseFloat(localStorage.getItem('scale')) || 1 : 1,
+    rotate: !isStudent ? parseInt(localStorage.getItem('rotate')) || 0 : 0
+  };
```
Comment (MasterOdin) [inline site/public/js/pdf/PDFAnnotateEmbedded.js]: can remove this change

File: site/public/js/pdf/PDFAnnotateEmbedded.js [hunk 4]
```diff
@@ -27,18 +21,25 @@ pdfjsLib.GlobalWorkerOptions.workerSrc = 'vendor/pdfjs/pdf.worker.min.js';
...
 //as a parameter.
-function render_student(gradeable_id, user_id, file_name, file_path, pdf_url) {
-    render(gradeable_id, user_id, "", file_name, file_path, 1, pdf_url)
+function render_student(gradeable_id, user_id, file_name, file_path, pdf_url, isStudent) {
+    render(gradeable_id, user_id, "", file_name, file_path, 1, pdf_url, isStudent )
 }
 
-function render(gradeable_id, user_id, grader_id, file_name, file_path, page_num, url = "") {
-    window.GENERAL_INFORMATION = {
+function render(gradeable_id, user_id, grader_id, file_name, file_path, page_num, url = "", isStudent = false) {
```
Comment (MasterOdin) [inline site/public/js/pdf/PDFAnnotateEmbedded.js]: revert this change

**Comment by mukul-kmr-jha** [comment]
Thanks for the detailed suggestions :).
Implemented the suggested changes and reverted the previous ones.

**Pull request merged**